portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,63 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Alexandre Lopes</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <header>
+      <div class="header">
+
+        <nav>
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+            </ul>
+        </nav>
+      </div>
+</header>
+
+<body>
+    <section class="home" id="home">
+        <div class="max-width">
+            <div class="home-content">
+                <div class="text-1">
+                    Hello World!
+                </div>
+                <div class="text-2">
+                    I'm Alexandre
+                </div>
+                <div class="text-3">
+                    Undergrad CS student
+                </div>
+                <a href="./images/AlexandreLopes_Resume.pdf" target="_blank" download" class="button-home">Downlaod Resume</a>
+            </div>
+        </div>
+    </section>
+    <div class="container" id="about">
+            <header>
+            <h1>About</h1>
+            </header>
+            <div class="about-content">
+             <p>Hello! My name is Alexandre Miguel. I am a college student (3.91 GPA, Bachelor of Science
```

Comment:
Please be consistent with indentation. Whether you use 2-space indentation or 4-space indentation. The closing tag should always be at the same level as the opening tag and tags within other tags should be further indented

Reply:
Will be. Thanks for the feedback!